src/CommonITILObject.php

```diff
@@ -123,7 +123,7 @@ public function loadUsers(): void
     {
         if (!empty($this->userlinkclass) && isset($this->fields['id'])) {
             $class = new $this->userlinkclass();
-            $this->lazy_loaded_users = $class->getActors($this->fields['id']);
+            $this->lazy_loaded_users = $class->getActors((int) $this->fields['id']);
```

Comment:
This (and other similar changes) should be changed on 10.0/bf branch

Reply:
No longer needed. 10.0 fix changes the `if` condition so `getActors` isn't called at all for new items.